index.html

```diff
@@ -45,6 +45,174 @@ <h1>
       <img src="images/Normal Button/Secondary/Disabled.png" alt="">
     </section>
   </section>
+  <section class="r_works">
+    <section class="title">
+      <p>My Recent Works</p>
+      <p class="s"></p>
+    </section>
+
+    <section class="works grid-container">
+        <section class="grid_1-item">
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
```

Comment:
- [OPTIONAL] Also just as I mentioned above, this is a title, so I'll suggest you change the tag of all the project titles in this work section to `h2 or h3` :+1: :wave:

Reply:
Ok Done